spec.md

```diff
@@ -375,14 +444,15 @@ set, the last (greatest) element is the furthest away.
 
 ### Bucket Index
 
-The bucket index can be computed using the following function:
+The index of bucket can be computed using the following function:
```

Comment:
Er, that's right.
Rationale for the change was, as noted in the commit message, that while reading first time I'd been confused  if it was "[The bucket]'s index" or "The [bucket index] of something else contained in bucket"? (though it was possible to derive from context).
I admit, this can be just my misunderstanding of grammar/semantics, as English is not my native